charts/spire/values.yaml

```
@@ -85,6 +97,20 @@ server:
       organization: Example
       common_name: example.org
 
+    upstreamAuthority:
+      certManager:
+        enabled: false
+        issuerName: "spire-ca"
+        issuerKind: "Issuer"
+        issuerGroup: "cert-manager.io"
+        namespace: "sandbox"
```

```suggestion
        namespace: cert-manager
```

I assume most people will by default deploy to this namespace.